measures/2018/measures-data.json

```diff
@@ -44669,7 +44669,6 @@
     "lastPerformanceYear": null,
     "isHighPriority": true,
     "isInverse": false,
-    "overallAlgorithm": "overallStratumOnly",
```

Comment:
Why did these `"overallAlgorithm": "overallStratumOnly",` lines get removed? Just want to understand :)